paasta_tools/cli/cmds/status.py

```diff
@@ -2407,9 +2407,13 @@ def paasta_status(args) -> int:
 
     with concurrent.futures.ThreadPoolExecutor(max_workers=20) as executor:
         tasks = [executor.submit(t[0], **t[1]) for t in tasks]  # type: ignore
-        for future in concurrent.futures.as_completed(tasks):  # type: ignore
-            return_code, output = future.result()
-            return_codes.append(return_code)
+        try:
+            for future in concurrent.futures.as_completed(tasks):  # type: ignore
+                return_code, output = future.result()
+                return_codes.append(return_code)
+        except KeyboardInterrupt:
+            executor._threads.clear()
+            concurrent.futures.thread._threads_queues.clear()
```

Comment:
should we re-raise here so that the return code isn't set by the `return max(return_codes)` below?

Reply:
No clue I got exit code 1 (which is what I was expecting) when I tested.

Reply:
my understanding is that we should be seeing 130 on a KeyboardInterrupt (e.g., that's what you get if you ^C something like `sleep 1000` and check the exit code)

Reply:
For some reason in my head KeyboardInterrupt exit with code 1. Thanks for the catch!!!